### ISSUE TITLE
Telerik.Web.UI.dll 2013.2.717.40

### ISSUE THREAD

**Issue by B8ightY** [dependency-check/DependencyCheck#2893]
Hi Jeremy,

I would like to reopen [this](https://github.com/jeremylong/DependencyCheck/issues/2726#issue-659059114) issue due problem with Telerik.Web.UI library in version 2013.2.717.40. I tried to check it with ODC multiple times with several versions of that library which are working as expected but in this specific version is no vulnerability found, despite of the CPE [exists](https://nvd.nist.gov/products/cpe/detail/663834?namingFormat=2.3&orderBy=CPEURI&keyword=cpe:2.3:a:telerik&status=FINAL).

**Here are some info:** 
1. ODC v6.0.2
2. dotnet v3.1.402
3. [odc.log](https://github.com/jeremylong/DependencyCheck/files/5378174/odc.log)
4. [dependency-check-report.zip](https://github.com/jeremylong/DependencyCheck/files/5378164/dependency-check-report.zip)

Can you please help me to find out what's causing that no vulnerability is found?

Thank you!

**Comment by jeremylong** [comment]
Any chance you could provide the JSON formatted report for a version of the control that identified the vulnerabilities and the JSON report for 2013.2.717.40?

**Comment by B8ightY** [comment]
I mentioned the link of the report for 2013.2.717.40 above in my question but I don't have the JSON one, sorry.

Anyway using JustDecompile tool I found out this .dll I scanned with ODC has assembly product of "Telerik.Web.UI" and the other ones in which the vulnerabilities are found have assembly product of "Progress® Telerik® UI for ASP.NET AJAX". So it seems to be not bug in your tool because the "Telerik.Web.UI" assembly product is not listed in the NVD and no CPE exists for that.

My apologies for the misunderstanding. I am closing this issue. Thank you for your support
